### ISSUE TITLE
empty Extent

### ISSUE THREAD

**Issue by visr** [rafaqz/Extents.jl#4]
I thought for a moment this returned a different type, but it's just `show`.

```
julia> Extent()
ExtentNamedTuple()
```

But it made me think, should we allow these to be created?
And what about tuples for a dimension that are not size 2?

```
julia> Extent(X=(2,3,4))
Extent(X = (2, 3, 4),)

julia> Extent(X=(2,))
Extent(X = (2,),)
```

Should we check for these in the inner constructor?

**Comment by rafaqz** [comment]
Thanks this is good feedback. We should only allow 2 tuples.

But Im not sure how to handle antimerideans. We could force ascending order in Tuples, but add a struct wrapper for that adds the wrapping point (e.g. 180).

**Comment by visr** [comment]
Yeah perhaps we should park the antimeridian for now, it's quite domain specific.

See #6.

Related to the 2 tuples, do you think it makes sense to add outer constructors that support AbstractVector and then run `extrema` on them to find the extents, such that this would work:

```
julia> Extent(X=[2,1,4])

julia> extrema([2,1,4])
(1, 4)
```
